CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.18.4 (Unreleased)
 
+- ApiGateway now provides control over the backing s3.Bucket created for `StaticRoute`s.  This is
+  useful for SinglePageApp scenarios that want to relevant Bucket values like `errorDocument` or
```

Comment:
I think you missed a word here, and it should be something like "useful for SinglePageApp scenarios that want to *control* relevant Bucket values like"

Reply:
yup yup